integrated_channels/sap_success_factors/transmitters/courses.py

```diff
@@ -0,0 +1,30 @@
+"""
+Class for transmitting course data to SuccessFactors.
+"""
+from __future__ import unicode_literals
+from integrated_channels.sap_success_factors.transmitters import BaseSuccessFactorsTransmitter
+
+
+class SuccessFactorsCourseTransmitter(BaseSuccessFactorsTransmitter):
+    """
+    This endpoint is intended to carry out an export of course data to SuccessFactors for a given Enterprise.
```

Comment:
We should probably be a little more specific about whether we are transmitting Course data or Course Run data